watchdog.go

```diff
@@ -70,6 +72,7 @@ func NewWatchdog(
 	w := &Watchdog{
 		PathToChildExecutable: pathToChildExecutable,
 		Args:                     cpOfArgs,
+		Starting:                  make(chan bool, 1),
```

Comment:
nit: run gofmt here. 

What editor do you use? 
[VS code has a go plugin.](https://robots.thoughtbot.com/writing-go-in-vim)
[Using go with emacs](http://dominik.honnef.co/posts/2013/03/writing_go_in_emacs/)
[Using go with vim](https://robots.thoughtbot.com/writing-go-in-vim)

Reply:
Atom, I downloaded one and turned on auto format but didn't turn on the plugin.. Fixed